### PR TITLE
Add Picture in Picture events in HTML5Video

### PR DESCRIPTION
**Pull request by brunacm-dev**
## Summary

This PR implements the picture in picture feature that can be useful for PiP Plugins

## Changes

- `html5_video.js`:
  - add handlers to picture-in-picture 
- `events.js`: 
  -  add events to handle EnterPIP and ExitPIP
- `container.js`: 
  - bind pip playback events to container events
- `playback.js`: 
  - add methods to pip mode

## How to test

1. There's an example in code comments that uses a simple button

### After this PR

pip

### CODE REVIEW

File: src/base/playback/playback.js [hunk 1]
```diff
@@ -23,10 +23,10 @@ import $ from 'clappr-zepto'
...
 export default class Playback extends UIObject {
   /**
-  * Determine if the playback does not contain video/has video but video should be ignored.
-  * @property isAudioOnly
-  * @type Boolean
-  */
+   * Determine if the playback does not contain video/has video but video should be ignored.
+   * @property isAudioOnly
+   * @type Boolean
+   */
```
Comment (arielfurtado) [inline src/base/playback/playback.js:29]: remove this white space

File: src/playbacks/html5_video/html5_video.js [hunk 2]
```diff
@@ -695,6 +730,15 @@ export default class HTML5Video extends Playback {
...
 
     this._ready()
+
+    // Note: Example of Picture-in-Picture
+    // const pipButton = document.createElement('button', { class: 'pip-button' })
+    // document.body.appendChild(pipButton)
+    // pipButton.innerHTML = 'Picture-in-Picture'
+    // pipButton.addEventListener('click', () => {
+    //   this.togglePIP()
+    // })
```
Comment (arielfurtado) [inline src/playbacks/html5_video/html5_video.js:740]: remove this unecessary comment

Comment (brunacm-dev) [reply src/playbacks/html5_video/html5_video.js]: Done in [Commit](https://github.com/clappr/clappr-core/pull/120/commits/7d8e32492c1d895c10d1b9200e151bcaf1c01e00)

File: src/base/playback/playback.js (hunk 1 above)
Comment (brunacm-dev) [reply src/base/playback/playback.js]: Done in [Commit](https://github.com/clappr/clappr-core/pull/120/commits/4182cd4e5e8bcec65dfbd25174874336999cd637)

**Review by pedrochamberlain — APPROVED** [review]

**Review by leaofelipe — APPROVED** [review]

**Pull request merged**